docs/developer-guide/repos.md

```diff
@@ -135,7 +135,7 @@ The other repos are used by one of the other repos above:
 * [node-png-joyent](https://github.com/TritonDataCenter/node-png-joyent): An ancient branch of node-png
 * [node-amqp-joyent](https://github.com/TritonDataCenter/node-amqp-joyent): An ancient branch of node-amqp
 * [node-imgmanifest](https://github.com/TritonDataCenter/node-imgmanifest): Node.js library for working with SmartOS image manifests
-* [sdc-fast-stream](https://github.com/TritonDataCenter/sdc-fast-stream): Stream event messages via node-fast.
+* [node-fast-stream](https://github.com/TritonDataCenter/node-fast-messages): Stream event messages via node-fast.
```

Comment:
The link text here should also be `node-fast-messages` instead of `node-fast-stream`.

Reply:
Fixed in 0b2be4d.